src/pages/join_us/join-us-form-section.tsx

```diff
@@ -0,0 +1,288 @@
+import React from 'react';
+import { IProps, IState } from '../../interfaces/JoinUsFormInterface';
+import Select from 'react-select';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as Yup from 'yup';
+
+const skillOptions = [
```

Comment:
Remove `skillOptions`. Use React select. I will look in to the react select issue